miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -358,7 +358,7 @@
         </c:forEach>
       </div>
     </c:if>
-    <div id="addRunNoteDialog" title="Create new Note"></div>
+    <div id="addRunNoteDialog" title="Add Container"></div>
```

Comment:
Wait...are add container and add note sharing the same dialog? Will this made add note wrong?